### DISCUSSION TITLE
Themes for Blazor?

### DISCUSSION THREAD

**Discussion by 274188A** [EasyAbp/EasyAbpGuide#16] [General]
Hi, just wondering if there any plans to build simple themes for blazor? 

Thanks

**Comment by gdlcf88** [comment]
https://blog.abp.io/abp/LeptonX-Theme-for-ABP-Framework-Alpha-Release

I believe the LeptonX is the best template at present, I am waiting for its Blazor version release.

**Reply by 274188A** [reply]
great thanks!

**Comment by gdlcf88** [comment]
And by the way, the [Entity UI](https://github.com/EasyAbp/Abp.EntityUi) module will help us easy to create cross-frontend management pages by some standardized JSON configurations. (Although it is only implemented in the MVC template)

**Reply by 274188A** [reply]
Nice! - look forward to trying that for sure
